Source/displayinfo/DisplayInfo.cpp

```diff
@@ -415,6 +415,43 @@ class DisplayInfo : public Core::IReferenceCounted {
         return _displayConnection->EDID(len, data);
     }
 
+    displayinfo_hdr_caps_t TvHDR() const
+    {
+        displayinfo_hdr_caps_t value = DISPLAYINFO_HDR_OFF;
+        Exchange::IHDRProperties::IHDRIterator* tvHdrSupport;
+        Exchange::IHDRProperties::HDRType hdr;
+
+        if ((_hdrProperties != nullptr) && (_hdrProperties->TVCapabilities(tvHdrSupport) == Core::ERROR_NONE) && (tvHdrSupport != nullptr)) {
+            //Count number of elements in list, if count > 0, then TV supports HDR
+            if (tvHdrSupport->Count()) {
```

Comment:
There should be no need to call this tvHdrSupport->Count() . The Next will simply return fals on the first call. This safes a cross process boundary call in case there are elements.

Reply:
ok